### PR TITLE
chore: update php version matrix on security and test workflows

### PR DESCRIPTION
**Pull request by jennifer-choo**
## Description
Update ci workflow to use php versions that are contained in the main composer.json file

Related issue: [JIRA_TICKET_NUMBER](LINK_TO_JIRA_TICKET)

## Before submitting (or marking as "ready for review")

+ [x] Does the pull request title follow the [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/) specification?
+ [ ] Have you performed a self-review of the code
+ [ ] Have you have added tests that prove the fix or feature is effective and working
+ [ ] Did you make sure to update any documentation relating to this change?

### CODE REVIEW

**Review by DariuszJerzewski — APPROVED** [review]

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -19,12 +19,12 @@ jobs:
   test:
     uses: dvsa/.github/.github/workflows/php-library-tests.yml@v3.2.3
     with:
-      php-versions: "[\"8.0\", \"8.1\"]"
+      php-versions: "[\"7.4\", \"8.0\", \"8.1\"]"
```
Comment (faggc) [inline .github/workflows/ci.yml:22]: Do we need php 7 here?

File: composer.json [hunk 2]
```diff
@@ -18,7 +18,7 @@
     "require-dev": {
         "phpunit/phpunit": "^9.5",
         "laminas/laminas-test": "^4.0.0",
-        "laminas/laminas-component-installer": "^3.2",
+        "laminas/laminas-component-installer": "^2.5",
```
Comment (faggc) [inline composer.json:21]: Downgrade?

File: .github/workflows/ci.yml (hunk 1 above)
Comment (jennifer-choo) [reply .github/workflows/ci.yml:22]: Just added it for coverage because the main composer.json file uses 7.4 as well

File: composer.json (hunk 2 above)
Comment (jennifer-choo) [reply composer.json:21]: Originally I just had 8.0 and 8.1 for the security and test ci workflows which were complaining about this version (2.5) being outdated so I updated it, now that I've added 7.4 it complains about the 3.2 version being too recent, the original version of this on the gitlab repo was 2.5 so I just reverted the version back

**Review by faggc — APPROVED** [review]

**Pull request merged**